tests/qunit/editor/test.js

```diff
@@ -22,6 +23,12 @@ export default class EditorTest extends EditorBase {
 				// eslint-disable-next-line no-console
 				console.log( `Done: ${ module } -> ${ name }` );
 			}
+
+			if ( QUnit.config.validateContainersAlive ) {
```

Comment:
Why not run the test always?

Reply:
It is possible we just need to increase the timeout you decide please 👍

Reply:
Handled.